config.go

```diff
@@ -11,17 +11,18 @@ import (
 
 // Config holds application configuration settings
 type Config struct {
-	Tel                string `yaml:"tel"`                // Our telephone number
-	Server             string `yaml:"server"`             // The TextSecure server URL
-	RootCA             string `yaml:"rootCA"`             // The TLS signing certificate of the server we connect to
-	ProxyServer        string `yaml:"proxy"`              // HTTP Proxy URL if one is being used
-	VerificationType   string `yaml:"verificationType"`   // Code verification method during registration (SMS/VOICE/DEV)
-	StorageDir         string `yaml:"storageDir"`         // Directory for the persistent storage
-	UnencryptedStorage bool   `yaml:"unencryptedStorage"` // Whether to store plaintext keys and session state (only for development)
-	StoragePassword    string `yaml:"storagePassword"`    // Password to the storage
-	LogLevel           string `yaml:"loglevel"`           // Verbosity of the logging messages
-	UserAgent          string `yaml:"userAgent"`          // Override for the default HTTP User Agent header field
-	AlwaysTrustPeerID  bool   `yaml:"alwaysTrustPeerID"`  // Workaround until proper handling of peer reregistering with new ID.
+	Tel                   string `yaml:"tel"`                   // Our telephone number
+	Server                string `yaml:"server"`                // The TextSecure server URL
+	RootCA                string `yaml:"rootCA"`                // The TLS signing certificate of the server we connect to
+	ProxyServer           string `yaml:"proxy"`                 // HTTP Proxy URL if one is being used
+	VerificationType      string `yaml:"verificationType"`      // Code verification method during registration (SMS/VOICE/DEV)
+	StorageDir            string `yaml:"storageDir"`            // Directory for the persistent storage
+	UnencryptedStorage    bool   `yaml:"unencryptedStorage"`    // Whether to store plaintext keys and session state (only for development)
+	StoragePassword       string `yaml:"storagePassword"`       // Password to the storage
+	LogLevel              string `yaml:"loglevel"`              // Verbosity of the logging messages
+	UserAgent             string `yaml:"userAgent"`             // Override for the default HTTP User Agent header field
+	AlwaysTrustPeerID     bool   `yaml:"alwaysTrustPeerID"`     // Workaround until proper handling of peer reregistering with new ID.
+	EnableMultiDeviceSync bool   `yaml:"enableMultiDeviceSync"` // Enable multi-device synchronization
 }
```

Comment:
Added this option because it was experimental but also can think of a few use cases where one might want to disable this. For example, your desktop was compromised and you want disable device syncing temporarily? or you deliberately want to ensure no SyncMessages are ever sent. This feature allows syncing of private data (contacts/groups) and perhaps one wants to completely disable this? You could always just unlink the device as well so not sure how useful it is to keep this option.